test/isUndefined.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+/* eslint-env mocha */
+
+const assert = require('assert')
+const { isUndefined } = require('../index')
+
+describe('isUndefined', () => {
+  it('should return true for undefined', () => {
+    assert.ok(isUndefined(undefined))
+  })
+
+  it('should return true for no param', () => {
+    assert.ok(isUndefined())
+  })
+})
```

Comment:
not sure if this is full coverage as I haven't run tests locally but we should add also case when it returns false

Reply:
you are right, I have added full coverage for falsy values